Plugins/NativeEngine/Source/NativeEngine.h

```diff
@@ -250,10 +251,13 @@ namespace Babylon
         void Bind(FrameBufferData* data)
         {
             m_boundFrameBuffer = data;
+            
+            const bool isNewFrameBuffer = m_boundFrameBuffer->ViewId == 0 && m_boundFrameBuffer != m_defaultBackBuffer;
+            const auto viewId = isNewFrameBuffer ? GetNewViewId() : m_boundFrameBuffer->ViewId;
```

Comment:
Why do this here and not at `FrameBufferData`'s creation time? The check on 255 is a bit odd.